src/pages/index.tsx

```diff
@@ -175,10 +175,10 @@ function Contacts() {
         <h2 id="contact">Contact</h2>
         <ul>
           <li>
-            <Link to="https://chat.fedoraproject.org/#/room/#packit:fedora.im">
+            <Link to="https://matrix.to/#/%23packit:fedora.im">
               #packit:fedora.im
             </Link>{" "}
-            (Element / Matrix)
+            (Matrix)
```

Comment:
```suggestion
            (Matrix - choose your app)
            <Link to="https://chat.fedoraproject.org/#/room/#packit:fedora.im">
              #packit:fedora.im
            </Link>{" "}
            (Matrix - Element app)            
```
We were wondering if having the old link can make life easier for users less used to matrix. So we were thinking of having them both.

Reply:
Thanks for this useful summary. We will discuss it in the team once more and I will let you know.

Reply:
I think when adding this to our docs, we may have just copied this from how [Fedora docs](https://docs.fedoraproject.org/en-US/project/communications/#_primary_channels) have it. So I don't see any big issue with this. I would probably still prefer to provide both links so people can choose.

Reply:
When you go to [https://matrix.to/#/#packit:fedora.im?web-instance[element.io]=chat.fedoraproject.org](https://matrix.to/#/#packit:fedora.im?web-instance%5Belement.io%5D=chat.fedoraproject.org), you can choose the _Element_ app (default choice) and there is _Open on chat.fedoraproject.org_ link which is the same as our "old" link. So IMHO it makes sense to have just one link.

Reply:
Exactly what Nikola said. Compared to the existing link, it's just one extra click to confirm client and server for users wishing to use Element web on Fedora server. :)

But honestly, please double check you want to use Fedora server as default there. I can explain how it all differs in more detail if you want.